target/linux/brcm63xx/dts/vr-3022eu.dts

```diff
@@ -0,0 +1,108 @@
+/dts-v1/;
```

Comment:
Please include licensing info for your DTS file, see
https://openwrt.org/submitting-patches#dts_checklist
e.g. a line like:
`// SPDX-License-Identifier: GPL-2.0-or-later OR MIT`
above the `/dts-v1/;`

Reply:
@mruzicka Are you still interested to get this in?